packages/generate/src/queries.ts

```diff
@@ -38,6 +45,21 @@ currently supported.`);
 
   console.log(`Detected schema directory: ${params.schemaDir}`);
   const matches = await getMatches(root, params.schemaDir);
+
+  const errs:string[] = [];
+  function printSummary(){
+    const passedCount = matches.length - errs.length
+    const summary = `\n\x1b[${errs.length > 0 ? '1;31m' : '1;32m'}${passedCount}/${matches.length} queries passed${errs.length ? ', '+errs.length+' failed'  : '!'}`
+    console.log(summary)
+    for (const err of errs) {
+        console.log('\x1b[22m')
+        console.error(err)
```

Comment:
Does this work as expected? `console.log` outputs to `stdout` and `console.error` outputs to `stderr`.

Reply:
I've fixed this with the latest commit.